packages/workflow/src/Interfaces.ts

```diff
@@ -549,7 +549,34 @@ export interface IGetNodeParameterOptions {
 	extractValue?: boolean;
 }
 
-export interface IExecuteFunctions {
+namespace ExecuteFunctions {
+	namespace NumberReturning {
+		export type NodeParameter = 'limit';
+	}
+
+	export type GetNodeParameterFn = {
+		// @TECH_DEBT: Refactor to remove this barely used overload
```

Comment:
can you create a ticket for this?

Reply:
Link: https://linear.app/n8n/issue/N8N-5632/refactor-to-remove-overload-with-generic-for-getnodeparameter